src/Auth0/Login/Traits/Auth0UserTrait.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Auth0\Login\Traits;
+
+trait Auth0UserTrait {
+
+    /**
+     * @var array
+     */
+    protected $userInfo;
+
+    /**
+     * @var string
+     */
+    protected $accessToken;
+
+    /**
+     * Get the unique identifier for the user.
+     *
+     * @return string|null
+     */
+    public function getAuthIdentifier() : ?string
+    {
+        return $this->userInfo['sub'] ?? null;
+    }
+
+    /**
+     * Get id field name.
+     *
+     * @return string
+     */
+    public function getAuthIdentifierName() : string
+    {
+        return 'id';
+    }
+
+    /**
+     * Get the password for the user.
+     *
+     * @return string|null
+     */
+    public function getAuthPassword() : ?string
```

Comment:
This is now a part of `Auth0JWTUser`. Before that returned nothing/void and now will always return null.